src/python-common/ceph/cephadm/images.py

```diff
@@ -0,0 +1,19 @@
+# Default container images -----------------------------------------------------
+DEFAULT_PROMETHEUS_IMAGE = 'quay.io/prometheus/prometheus:v2.51.0'
+DEFAULT_NODE_EXPORTER_IMAGE = 'quay.io/prometheus/node-exporter:v1.7.0'
+DEFAULT_NVMEOF_IMAGE = 'quay.io/ceph/nvmeof:1.2.17'
+DEFAULT_LOKI_IMAGE = 'quay.io/ceph/loki:3.0.0'
+DEFAULT_PROMTAIL_IMAGE = 'quay.io/ceph/promtail:3.0.0'
+DEFAULT_ALERT_MANAGER_IMAGE = 'quay.io/prometheus/alertmanager:v0.27.0'
```

Comment:
I merged another PR that added some new import lines and didn't rebase this before testing, resulting in
```
2024-10-21T03:12:11.144 INFO:teuthology.orchestra.run.smithi133.stderr:Traceback (most recent call last):
2024-10-21T03:12:11.145 INFO:teuthology.orchestra.run.smithi133.stderr:  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
2024-10-21T03:12:11.151 INFO:teuthology.orchestra.run.smithi162.stderr:Traceback (most recent call last):
2024-10-21T03:12:11.151 INFO:teuthology.orchestra.run.smithi162.stderr:  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
2024-10-21T03:12:11.156 INFO:teuthology.orchestra.run.smithi133.stderr:    return _run_code(code, main_globals, None,
2024-10-21T03:12:11.156 INFO:teuthology.orchestra.run.smithi133.stderr:  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
2024-10-21T03:12:11.157 INFO:teuthology.orchestra.run.smithi133.stderr:    exec(code, run_globals)
2024-10-21T03:12:11.157 INFO:teuthology.orchestra.run.smithi133.stderr:  File "/tmp/tmpzfwsq3gn.cephadm.build/app/__main__.py", line 141, in <module>
2024-10-21T03:12:11.157 INFO:teuthology.orchestra.run.smithi133.stderr:  File "/tmp/tmpzfwsq3gn.cephadm.build/app/cephadmlib/runscripts.py", line 10, in <module>
2024-10-21T03:12:11.157 INFO:teuthology.orchestra.run.smithi133.stderr:  File "/tmp/tmpzfwsq3gn.cephadm.build/app/cephadmlib/container_types.py", line 11, in <module>
2024-10-21T03:12:11.157 INFO:teuthology.orchestra.run.smithi133.stderr:ImportError: cannot import name 'DEFAULT_ALERT_MANAGER_IMAGE' from 'cephadmlib.constants' (/home/ubuntu/cephtest/cephadm/cephadmlib/constants.pyc)
2024-10-21T03:12:11.159 INFO:teuthology.orchestra.run.smithi162.stderr:    return _run_code(code, main_globals, None,
2024-10-21T03:12:11.159 INFO:teuthology.orchestra.run.smithi162.stderr:  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
2024-10-21T03:12:11.160 INFO:teuthology.orchestra.run.smithi162.stderr:    exec(code, run_globals)
2024-10-21T03:12:11.160 INFO:teuthology.orchestra.run.smithi162.stderr:  File "/tmp/tmpzfwsq3gn.cephadm.build/app/__main__.py", line 141, in <module>
2024-10-21T03:12:11.160 INFO:teuthology.orchestra.run.smithi162.stderr:  File "/tmp/tmpzfwsq3gn.cephadm.build/app/cephadmlib/runscripts.py", line 10, in <module>
2024-10-21T03:12:11.160 INFO:teuthology.orchestra.run.smithi162.stderr:  File "/tmp/tmpzfwsq3gn.cephadm.build/app/cephadmlib/container_types.py", line 11, in <module>
2024-10-21T03:12:11.160 INFO:teuthology.orchestra.run.smithi162.stderr:ImportError: cannot import name 'DEFAULT_ALERT_MANAGER_IMAGE' from 'cephadmlib.constants' 
```
need to rebase this and fix it up